protocol/src/main/java/org/geysermc/mcprotocollib/network/crypt/PacketEncryption.java

```diff
@@ -2,24 +2,9 @@
 
 /**
  * An interface for encrypting packets.
+ * The outputLength should always be the same as the inputLength.
```

Comment:
Can we add a comment here, that this is vanilla behavior?

Reply:
That's the behaviour that the algorithm vanilla uses has. Ig I can add a comment saying that this is a trait of the algo vanilla uses.